src/libraries/System.ComponentModel.TypeConverter/tests/ILLink.Descriptor.xml

```diff
@@ -0,0 +1,21 @@
+<linker>
```

Comment:
Nit: a comment at the top of this file to give visibility into what this file does/gets consumed  and why it is needed

Reply:
Will do.